bin/main.rb

```diff
@@ -1,11 +1,14 @@
 require './lib/game_logic'
```

Comment:
- [ ] **OPTIONAL** Using require_relative for files in your directory instead of require is a ruby convention.

> "Require should be used for external files, like gems, while require_relative should be used for referring to files within your directory."